### PR TITLE
Closes #1 - Create initial index.html markup

### LINKED ISSUE #1: Create initial index.html markup

### PR DESCRIPTION
**Pull request by koldovsky**

### CODE REVIEW

**Review by CrazyRoka — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
+</html>
```
Comment (RokaUA) [inline index.html]: Please add empty line

Comment (markiyanshvets) [inline index.html]: Please add empty line

Comment (TarasKvasnytsya) [inline index.html]: Add empty line

**Review by markiyanshvets — CHANGES_REQUESTED** [review]

**Review by TarasKvasnytsya — CHANGES_REQUESTED** [review]
new comment

File: index.html (hunk 1 above)
Comment (CrazyRoka) [inline index.html]: Please add empty line

Comment (IrynaPanchuk97) [inline index.html]: Please add empty line

**Review by CrazyRoka — CHANGES_REQUESTED** [review]

**Review by danylo-dudok — CHANGES_REQUESTED** [review]
Add empty line

**Review by IrynaPanchuk97 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (maksym-yaremko) [inline index.html]: Please add empty line

**Review by maksym-yaremko — CHANGES_REQUESTED** [review]

**Review by yuriimochulskyi — CHANGES_REQUESTED** [review]
Please add empty line

**Review by danylo-dudok — APPROVED** [review]

**Review by maksym-yaremko — APPROVED** [review]

**Review by TarasKvasnytsya — APPROVED** [review]

**Review by IrynaPanchuk97 — APPROVED** [review]

**Review by CrazyRoka — APPROVED** [review]

**Review by yuriimochulskyi — APPROVED** [review]

**Review by markiyanshvets — APPROVED** [review]

**Pull request merged**